src/main/scala/mrtjp/projectred/expansion/TileAutoCrafter.scala

```diff
@@ -171,7 +171,7 @@ class TileAutoCrafter
   def containsEnoughResource(item: ItemKey, amount: Int): Boolean = {
     val eq = new ItemEquality
     eq.matchMeta = !item.makeStack(0).isItemStackDamageable
-    eq.matchNBT = false
+    eq.matchNBT = true
     eq.matchOre = currentRecipe.isInstanceOf[ShapedOreRecipe] || currentRecipe
       .isInstanceOf[ShapelessOreRecipe]
```

Comment:
The Issue here is that you also need to set matchOre to false, as PamsHarvest craft and many other mods register the recipe as [ShapelessOreRecipe ](https://github.com/GTNewHorizons/harvestcraft/blob/64993019f165899004b26e0ac03d0eba1911afbc/src/main/java/com/pam/harvestcraft/PamFoodRecipes.java#L89)( even when not using any items with OreDict matching in the recipe) 

https://github.com/GTNewHorizons/MrTJPCore/blob/cceaed84c98c34eea7d402a15264278fc78a0b0b/src/main/scala/mrtjp/core/item/itemequality.scala#L36-L40

There is an early return for the matchOre